meta1v2/meta1_prefixes.c

```diff
@@ -78,6 +100,22 @@ meta1_prefixes_manage_all(struct meta1_prefixes_set_s *m1ps)
 
 /* NS operations ------------------------------------------------------------ */
 
+static guint8*
+_cache_from_all_m0l(const GSList *l)
+{
+	guint8 *result = g_malloc0(8192);
```

Comment:
Where does `8192` come from?

Reply:
The meta0 size is 64ki bits, 8ki bytes.
8192 is the size of the bitmap for managed prefixes.